### PR TITLE
Better IA structure and secondary navigation

### PR DESCRIPTION
**Pull request by dcwalk**
Hey, I'm feeling a little indecisive sorry for springing this-- but I think our website structure is a little confusing. I did a (cribbed) cleanup. Unfortunately I didn't keep this PR super clean and a couple commits are link fixes and an update to my bio.

Welcome your thoughts! 

screen shot 2018-05-13 at 9 33 27 pm

(basically: cut out the 2018 structure until the event ends, this is the IA:

```
. Home Splash (Registration)
├── About ✔︎
│   ├── 2017  ✔︎
│   └── Code of Conduct  ✔︎
├── 2018   ✘ (same as HOME until 2018 ends)
├── Yami-ichi 
├── Conference 
├── Sprints ✔︎
└── Visiting Toronto ✔︎
```

### CODE REVIEW

File: conference.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ endDate: 2018-07-15
 venueName: "Mozilla Community Space"
 location: "Suite 500, 366 Adelaide Street"
 locationLink: https://osm.org/go/ZX6BjZ50h?m=
+redirect_from:
+  - "/2018/conferences"
```
Comment (garrying) [inline conference.md]: Should this be `/2018/conference`?

File: sprints.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ endDate: 2018-07-18
 venueName: "Semaphore Demo Room"
 location: "Room 417, Claude T. Bissell, 140 St. George Street"
 locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+redirect_from:
+  - "/2018/sprint"
```
Comment (garrying) [inline sprints.md]: `/2018/sprints`?

**Review by garrying — COMMENTED** [review]
I think removing the `/2018/` is an improvement along with moving `2017`.  I've added a few adjustments for spacing and getting the active states to show on secondary pages.

**Comment by dcwalk** [comment]
thanks @garrying ! made the changes to the borked redirects (I think I muddled them in my mind) and also caught a missing word in the about copy.

Looks good to me!

**Comment by dcwalk** [comment]
Can we merge @garrying ?

**Comment by garrying** [comment]
Yeah. Squash and merging

**Comment by garrying** [comment]
@dcwalk added `data-proofer-ignore` in https://github.com/ournetworks/ournetworks.ca/commit/f76126faf1d9427f7edbc364cf9058c177c6ff1b

**Comment by dcwalk** [comment]
thanks! I forgot, SSL cert misconfig :((

**Pull request merged**